company-box-icons.el

```diff
@@ -226,5 +228,14 @@ specification.md#completion-request-leftwards_arrow_with_hook.")
   (when (get-text-property 0 'yas-annotation candidate)
     'Template))
 
+(defun company-box-icons-resize (size)
+  "Set icons size in pixels."
+  (interactive "nIcon size in pixels: ")
+  (mapc (lambda (icon)
```

Comment:
Shouldn't it setq company-box-icons-images?

Reply:
Strictly speaking - yes. The code won't work if the list contains nil elements.